flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ShowPartitionsOperation.java

```diff
@@ -66,10 +66,6 @@ public CatalogPartitionSpec getPartitionSpec() {
         return partitionSpec;
     }
 
-    public String getDefaultPartitionName() {
```

Comment:
Why delete this getter here?

Reply:
We won't need it any more since we can get `defaultPartitionName` directly in method `ShowPartitionsOperation#execute`.